render-test/main.cpp

```diff
@@ -19,6 +19,7 @@
 #define ANSI_COLOR_LIGHT_GRAY "\x1b[90m"
 #define ANSI_COLOR_RESET      "\x1b[0m"
 
+#if !defined(SANITIZE)
```

Comment:
@pozdnyakov this disables memory probing on sanitized builds, if we would add tests with memory metrics, they would fail.

Reply:
We cannot enable it on sanitize bots, because the sanitizers also replace the `new` operators. But the point of these bots is not to collect metrics.